### PR TITLE
fix culture name

### PR DESCRIPTION
**Pull request by AlexeyKovalkovvv**

### CODE REVIEW

**Review by sixg — COMMENTED** [review]
Need to discuss this with Yan

File: Splitit/Paymentmethod/Helper/Data.php [hunk 1]
```diff
@@ -475,9 +475,11 @@ public function getSplititSupportedCultures() {
...
 
 		$decodedResult = $this->jsonHelper->jsonDecode($getSplititSupportedCultures);
-		if (isset($decodedResult["ResponseHeader"]["Succeeded"]) && $decodedResult["ResponseHeader"]["Succeeded"] == 1 && count($decodedResult["SupportedCultures"])) {
-			return $decodedResult["SupportedCultures"];
-		}
+		if (isset($decodedResult["ResponseHeader"]["Succeeded"]) && $decodedResult["ResponseHeader"]["Succeeded"] == 1 && count($decodedResult["Cultures"])) {
+            return array_map(function($culture) {
+                return $culture['CultureName'];
+            }, $decodedResult["Cultures"]);
+        }
```
[the remark this answers is not in the document]
Comment (AlexeyKovalkovvv) [reply Splitit/Paymentmethod/Helper/Data.php:482]: Maybe yes, but I don't have anything to compare how it was before. Actually we a **SupportedCultures** in a response, but it contains list of cultures and it looks like `["English", "Russian", "Italian", etc]`

**Review by sixg — CHANGES_REQUESTED** [review]
Please raise this PR into staging instead of master

**Review by sixg — CHANGES_REQUESTED** [review]
PR to be raised for staging instead of master

**Pull request closed without merging**